libcxx/test/std/utilities/memory/util.smartptr/util.smartptr.shared/util.smartptr.shared.cmp/cmp_nullptr.pass.cpp

```diff
@@ -87,5 +87,38 @@ int main(int, char**)
   assert((nullptr <=> p2) == std::strong_ordering::equivalent);
 #endif
 
+#if TEST_STD_VER > 14
```

Comment:
nowadays we typically use this style.
```suggestion
#if TEST_STD_VER >= 17
```
Please apply to similar cases.

Reply:
Updated.